daysCountdown@KopfDesDaemons/files/daysCountdown@KopfDesDaemons/metadata.json

```
@@ -0,0 +1,7 @@
+{
+    "uuid": "daysCountdown@KopfDesDaemons",
+    "name": "Days Countdown",
+    "description": "Countdown of the remaining days to a date.",
+    "version": "1.0",
+    "max-instances": "50"
```

```suggestion
    "max-instances": "-1"
```

If you wish for it to be unlimited, this is the cleaner configuration.